examples/java8/src/main/java/org/apache/beam/examples/advanced/subprocess/kernel/SubProcessCommandLineArgs.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.examples.advanced.subprocess.kernel;
+
+import com.google.common.collect.Lists;
+
+import java.util.List;
+
+import org.apache.beam.sdk.coders.AvroCoder;
+import org.apache.beam.sdk.coders.DefaultCoder;
+
+
+
+/**
+ * Parameters to the sub-process, has tuple of ordinal position and the value.
+ */
+@DefaultCoder(AvroCoder.class)
```

Comment:
unless there is a PCollection<SubProcessCommandLineArgs> a Coder is not necessary

Reply:
Fixed